.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'release-v')
```

Comment:
How about detecting a release by version tag existence rather than using branch name?
e.g.) 
`pkg_version=$(get_version.sh)`
if `git tag -l $pkg_version` is empty, create new tag